packages/bot-web-ui/src/components/quick-strategy.jsx

```diff
@@ -62,77 +64,141 @@ const QuickStrategyForm = ({
-                                                selected_trade_type.icon && (
-                                                    <span className='quick_strategy__trade-type--icon'>
-                                                        <IconTradeTypes type={selected_trade_type.icon[0]} />
-                                                        <IconTradeTypes type={selected_trade_type.icon[1]} />
-                                                    </span>
-                                                )
-                                            }
-                                        />
+                                        <>
+                                            {!is_mobile ? (
+                                                <Autocomplete
+                                                    {...field}
+                                                    autoComplete='off'
+                                                    className='quick-strategy__dropdown quick-strategy__leading'
+                                                    type='text'
+                                                    label={localize('Trade type')}
+                                                    list_items={trade_type_dropdown}
+                                                    onHideDropdownList={() => {
+                                                        onHideDropdownList(
+                                                            'trade-type',
+                                                            values[field.name],
+                                                            setFieldValue
+                                                        );
+                                                    }}
+                                                    onItemSelection={({ value }) => {
+                                                        onChangeDropdownItem('trade-type', value, setFieldValue);
+                                                    }}
+                                                    onScrollStop={() => onScrollStopDropdownList('trade-type')}
+                                                    leading_icon={
+                                                        selected_trade_type.icon && (
+                                                            <span className='quick_strategy__trade-type--icon'>
+                                                                <IconTradeTypes type={selected_trade_type.icon[0]} />
+                                                                <IconTradeTypes type={selected_trade_type.icon[1]} />
+                                                            </span>
+                                                        )
+                                                    }
+                                                />
+                                            ) : (
+                                                <Dropdown
```

Comment:
Ditto here, should use SelectNative instead.

Reply:
Same as above.